third_party/libprotobuf_mutator.BUILD

```diff
@@ -0,0 +1,20 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
```

Comment:
So I believe this is largely borrowed/inspired by another project.

Rather than just putting it into a "third_party" directory, instead we probably want the directory structure to look like this:
third_party/(other-project-name)/(file)
third_party/(other-project-name)/LICENSE (from the other project)
third_party/(other-project-name)/README.md (brief readme with a link to the other project).

Reply:
Can you point to me the instruction to do so?